apps/hash-graph/libs/store/src/account/mod.rs

```diff
@@ -41,50 +51,53 @@ pub struct InsertWebIdParams {
     pub owner: WebOwnerSubject,
 }
 
+#[derive(Debug, Error)]
+#[error("Could not query web")]
+pub struct QueryWebError;
+
 /// Describes the API of a store implementation for accounts.
-#[async_trait]
 pub trait AccountStore {
     /// Inserts the specified [`AccountId`] into the database.
     ///
     /// # Errors
     ///
     /// - if insertion failed, e.g. because the [`AccountId`] already exists.
-    async fn insert_account_id(
+    fn insert_account_id(
```

Comment:
why `Result<.., Report<..>>` not `error_stack::Result`?

Reply:
I mentioned this before somewhere: I find `error_stack::Result` quite annoying as it's not visible from looking at the code that we deal with `Report` here.

Reply:
I agree, maybe we can come up with a better name to signal that, because I always find `Result<T, Report<C>>` quite cumbersome to write^^